pkg/agent/ops.go

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright © 2021 The LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package agent
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/litmuschaos/litmusctl/pkg/apis"
+	"github.com/litmuschaos/litmusctl/pkg/k8s"
+	"github.com/litmuschaos/litmusctl/pkg/types"
+	"github.com/litmuschaos/litmusctl/pkg/utils"
+)
+
+func PrintExistingAgents(agent apis.AgentData) {
+	fmt.Println("🚫 Agent with the given name already exists.")
+	// Print agent list if existing agent name is entered twice
+	fmt.Print("\n📘 Connected agents list -----------\n\n")
+
+	for i := range agent.Data.GetAgent {
+		fmt.Println("-", agent.Data.GetAgent[i].AgentName)
+	}
+
+	fmt.Println("\n-------------------------------------")
+
+	fmt.Println("❗ Please enter a different name.")
+}
+
+// GetProject display list of projects and returns the project id based on input
+func GetProjectID(u apis.ProjectDetails) string {
+	var pid int
+	fmt.Println("\n✨ Projects List:")
+	for index := range u.Data.GetUser.Projects {
+		fmt.Printf("%d.  %s\n", index+1, u.Data.GetUser.Projects[index].Name)
+	}
+
+repeat:
+	fmt.Print("\n🔎 Select Project: ")
+	fmt.Scanln(&pid)
+
+	for pid < 1 || pid > len(u.Data.GetUser.Projects) {
+		fmt.Println("❗ Invalid Project. Please select a correct one.")
+		goto repeat
+	}
+
+	return u.Data.GetUser.Projects[pid-1].ID
+}
+
+// GetMode gets mode of agent installation as input
+func GetModeType() string {
+repeat:
+	var mode = 1
+	fmt.Println("\n🔌 Installation Modes:\n1. Cluster\n2. Namespace")
+	fmt.Print("\n👉 Select Mode [", utils.DefaultMode, "]: ")
+	fmt.Scanln(&mode)
+
+	if mode == 1 {
+		return "cluster"
+	}
+
+	if mode == 2 {
+		return "namespace"
+	}
+
+	if mode != 1 || mode != 2 {
```

Comment:
there is a reviewdog error here...can you check that

Reply:
fixed